packages/core/src/exchanges/subscription.ts

```diff
@@ -52,13 +52,14 @@ export interface SubscriptionExchangeOpts {
   // This has been modelled to work with subscription-transport-ws
   // See: https://github.com/apollographql/subscriptions-transport-ws#requestoptions--observableexecutionresult-returns-observable-to-execute-the-operation
   forwardSubscription: SubscriptionForwarder;
-}
 
-const isSubscriptionOperation = (operation: Operation) =>
-  operation.operationName === 'subscription';
+  /** This flag may be turned on to allow your subscriptions-transport to handle all operation types */
+  enableAllOperations?: boolean;
```

Comment:
Why don't we use filter function instead? There might be cases (I cannot think of any) where someone wants to pass only subscriptions and mutation via subscription exchange.

Reply:
I don't think it makes sense to anticipate use-cases that we can't imagine. To a certain extent we tell people that "forking" our exchanges or making more specific ones is fine.

So overall we only implement what we can see as a use-case right now, instead of bloating our API surface 👍  Hope that makes sense to you though